packages/liveblocks-react/src/room.tsx

```diff
@@ -1311,8 +1309,16 @@ function useCommentsErrorListener<M extends BaseMetadata>(
 function useCreateThread<M extends BaseMetadata>(): (
   options: CreateThreadOptions<M>
 ) => ThreadData<M> {
+  return useCreateRoomThread(useRoom().id);
+}
+
+/**
+ * @private
+ */
+function useCreateRoomThread<M extends BaseMetadata>(
+  roomId: string
+): (options: CreateThreadOptions<M>) => ThreadData<M> {
```

Comment:
You'll notice that these room hooks haven't changed much. I only refactored them into a simpler hook that accepts a room id and does all the same things that it used to do previously. These simpler hooks are also used by our Comment and Composer component so that they can be used outside of RoomProvider.